src/inet/http_server.c

```diff
@@ -138,7 +145,7 @@ static int read_request(struct http_server_t *self_p,
                         struct http_server_request_t *request_p)
 {
     int res;
-    char buf[128];
```

Comment:
Yeah let's fix the logic in ``read_header_line()`` so it does not write outside the buffer. Read into the buffer, and when it is full just ignore the rest of the data.